models/issue.go

```diff
@@ -1076,6 +1076,7 @@ type IssuesOptions struct {
 	ExcludedLabelNames []string
 	SortType           string
 	IssueIDs           []int64
+	NotInProjectID     int64
```

Comment:
Hmm... not a fan of this name: `NotInProjectID`

```suggestion
	ExcludeProjectID   int64
```